filament/include/filament/RenderableManager.h

```diff
@@ -129,52 +295,132 @@ class UTILS_PUBLIC RenderableManager : public FilamentAPI {
         };
     };
 
-    // destroys this component from the given entity
+    /**
+     * Destroys the renderable component in the given entity.
+     */
     void destroy(utils::Entity e) noexcept;
 
+    /**
+     * Changes the bounding box used for frustum culling.
+     *
+     * See also Builder::boundingBox() and RenderableManager::getAxisAlignedBoundingBox().
+     */
     void setAxisAlignedBoundingBox(Instance instance, const Box& aabb) noexcept;
 
-    // See View::setVisibleLayers
+    /**
+     * Changes the visibility bits.
+     *
+     * See also Builder::layerMask() and View::setVisibleLayers().
+     */
     void setLayerMask(Instance instance, uint8_t select, uint8_t values) noexcept;
 
+    /**
+     * Changes the coarse-level draw ordering.
+     *
+     * See also Builder::priority().
+     */
     void setPriority(Instance instance, uint8_t priority) noexcept;
+
+    /**
+     * Changes whether or not the renderable casts shadows.
+     *
+     * See also Builder::castShadows().
```

Comment:
Maybe use `\see` for these "see also" blurbs? Not what that will look like rendered..

Reply:
Note that Doxygen automatically makes hyperlinks for these.

Reply:
Done.  Using \see makes a special section that arguably looks nicer.